packages/lib/src/components/internal/Address/Address.tsx

```diff
@@ -98,6 +98,7 @@ export default function Address(props: AddressProps) {
                 specifications={specifications}
                 maxlength={getMaxLengthByFieldAndCountry(countrySpecificFormatters, fieldName, data.country, true)}
                 trimOnBlur={true}
+                disabled={disablingTrigger && disablingTrigger !== fieldName}
```

Comment:
Minor thing and personal opinion here: 

I find this new property `disablingTrigger` on `Address` confusing to understand.  It is used only by the Card component, and can be hard for other devs that dig into the `Address` to understand what it implies. I am wondering if we can change it in a way which can be used by other components later on as well and can be easier to understand.

From what I understand, we need to disable all fields except the one that is currently focused. What if we provide a property like `enabledFields: string[]` which can be set one or more enabled fields - if nothing is set, then all fields are enabled by default.

In this way, we would pass to address the disablingTrigger:
```js
<Address
  {...props}
  enabledFields={disablingTrigger && [disablingTrigger]}
/>
```

I think doing this way it is more clear what is happening and might be reused in the future on other payment methods. Let me know your thoughts!

Reply:
I understand your concern. I found naming this particular property and some of the other props & functions required for this PR quite difficult.

My issue with your suggested `enabledFields` prop is that it then requires a knowledge form the parent of _all_ the fields in a component i.e it has to list them all in order for them to be enabled.

What if we had a `disabledFields` prop - which is more in line with what I'm already doing, namely just stating the field to exclude (rather than listing all the fields that need to be _included_)?

Reply:
Fair enough - I totally agree with the `disabledFields` property 👍  But how are you going to implement on this case which you have only the field name that should be enabled? You will need to set all other fields that needs to be disabled, right? Is it possible here?

Reply:
I'll figure it out. :-)
For now I'm merging this - 'cos I want this new functionality in place.
I'll put the new, more minor, change in a new PR